context/AuthContextProvider.tsx

```diff
@@ -54,6 +56,14 @@ export const AuthContextProvider = ({
     setIsSignedIn(true);
   }, [user]);
 
+  useMemo(() => {
+    if (isSignedIn) {
+      if (adminRoutes.includes(pathname)) {
+        !isSuperAdmin && router.push('/league/all');
```

Comment:
The homepage should redirect them to league/all already. Once this is set to the real homepage, we won't need to modify this again.
```suggestion
        !isSuperAdmin && router.push('/');
```

Reply:
@shashilo tried this out and nope I am not redirected automatically.

Reply:
done ✅